### DISCUSSION TITLE
Overseer - Radarr/Sonarr Settings - Disable Arrs?

### DISCUSSION THREAD

**Discussion by townsmcp** [itsToggle/plex_debrid#524] [Q&A] [answered]
Hey all. 
I have been looking all over the net and this GitHub repo but so far not found what I am looking for. 

So I have had Plex, Petio and the Arrs working well for a long time. Found plex_debrid yesterday so have it a go setting it all up (including switching to Overseer). It is all working fine so far but my question is, do I need to add the Arrs config into Overseer still or will that just cause any requested media to download with the Arrs and also with plex_debrid? Is it best just to delete the Arrs config from Overseer?

**Comment by Tw1zT3d2four7** [comment, marked as answer]
yup delete no need for them with the plex debrid and rclone setup.
